### ISSUE TITLE
[Content styleguide] Content doesn’t align with style guidance in VA.gov content style guide. (05.08.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#68707]
Pending location confirmation from @shiragoodman 

#### VFS team name
Contention Classification

#### VFS product name
Disability Comp Application (21-526EZ)

#### VFS feature name
update suggested conditions list

#### Point of Contact/Reviewers
Brian Smolensky - @BSmolensky - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content doesn’t align with style guidance in VA.gov content style guide.

### Issue Details
Disability Rating Info BOx: The first sentence in this info box contains a typo. There should be a comma after the conditional introductory clause "For each disability."

### VA.gov Experience Standard
[Category Number 05, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Please put a comma after the opening clause. 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by RakshindaAslam** [comment]
Need a screenshot of the actual issue.

**Comment by christinec-fftc** [comment]
is this the issue here on the introduction page? [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMEFiQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ffbfe691ea264ada3300c6b8c917659725b60dea/image.png)

**Comment by shiragoodman** [comment]
@BSmolensky could I please ask you to take a look at this and confirm the question above?

**Comment by BSmolensky** [comment]
@shiragoodman - It seems that I don't have permission to view the image.

**Comment by BSmolensky** [comment]
@shiragoodman , @RakshindaAslam @christinec-fftc: The arrow in the image is pointing to the correct spot. Sorry for the delay in responding.
